src/Sdk/DTPipelines/workflow-v1.0.json

```diff
@@ -79,6 +79,30 @@
       "string": {}
     },
 
+    "composite-outputs": {
```

Comment:
We also have to modify workflow.json because when we EvaluateCompositeOutputs in the outputs handler, the PipelineEvaluator uses workflow_1.0.json for evaluating tokens.

Reply:
In the CompositeOutputHandler, 

we do:
```
var evaluator = ExecutionContext.ToPipelineTemplateEvaluator();
                DictionaryContextData actionOutputs = evaluator.EvaluateCompositeOutputs(Data.Outputs, ExecutionContext.ExpressionValues, ExecutionContext.ExpressionFunctions);
```

This ExecutionContext will generate a template evaluator using the workflow yaml since in PipelineTemplateSchemaFactory, GetSchema will always return workflow yaml:
```
public static TemplateSchema GetSchema()
        {
            if (s_schema == null)
            {
                var assembly = Assembly.GetExecutingAssembly();
                var json = default(String);
                using (var stream = assembly.GetManifestResourceStream("GitHub.DistributedTask.Pipelines.ObjectTemplating.workflow-v1.0.json"))
                using (var streamReader = new StreamReader(stream))
                {
                    json = streamReader.ReadToEnd();
                }

                var objectReader = new JsonObjectReader(null, json);
                var schema = TemplateSchema.Load(objectReader);
                Interlocked.CompareExchange(ref s_schema, schema, null);
            }

            return s_schema;
        }
```

Reply:
I tried moving the code for GetSchema over to the CompositeActionOutputHandler but Assembly.GetExecutingAssembly() won't get the right directory for loading "GitHub.Runner.Worker.action_yaml.json"

Another possibility is to store the _actionManifestSchema (from ActionManifestManager) as a static variable in the ExecutionContext but since the ExecutionContext in the output handler is IExecutionContext, you can't access it unfortunately. 

I think it makes the most amount of sense to refactor this during abstraction.

Reply:
or we could just store it in our ExecutionContext via LoadAction